### PR TITLE
[Admin] Make pagination accessible

### PR DESCRIPTION
**Pull request by waiting-for-dev**
## Summary

We label the pagination icon wrappers with "aria-label", while hiding the icon itself to screen readers. We also mark as "aria-disable"'d when the first/last page is reached.

## Checklist

Check out our [PR guidelines](https://github.com/solidusio/.github/blob/master/CONTRIBUTING.md#pull-request-guidelines) for more details.

The following are mandatory for all PRs:

- [x] I have written a thorough PR description.
- [x] I have kept my commits small and atomic.
- [x] [I have used clear, explanatory commit messages](https://github.com/solidusio/.github/blob/main/CONTRIBUTING.md#writing-good-commit-messages).

The following are not always needed:

- 📖 I have updated the README to account for my changes.
- 📑 I have documented new code [with YARD](https://www.rubydoc.info/gems/yard/file/docs/Tags.md).
- 🛣️ I have opened a PR to update the [guides](https://github.com/solidusio/edgeguides).
- ✅ I have added automated tests to cover my changes.
- 📸 I have attached screenshots to demo visual changes.

### CODE REVIEW

File: admin/app/components/solidus_admin/base_component.rb [hunk 1]
```diff
@@ -14,7 +14,7 @@ class BaseComponent < ViewComponent::Base
     def icon_tag(name, **attrs)
       href = image_path("solidus_admin/remixicon.symbol.svg") + "#ri-#{name}"
       tag.svg(
-        class: attrs[:class],
```
Comment (kennyadsl) [inline admin/app/components/solidus_admin/base_component.rb:17]: This fix is not related, right? What about using another PR, just for documentation?

**Review by kennyadsl — APPROVED** [review]
Left a non-blocker. Thanks!

File: admin/app/components/solidus_admin/base_component.rb (hunk 1 above)
Comment (kennyadsl) [reply admin/app/components/solidus_admin/base_component.rb:17]: Just saw it's part of https://github.com/solidusio/solidus/pull/5268, so I guess it's here only until that one will be merged 👍

Comment (waiting-for-dev) [reply admin/app/components/solidus_admin/base_component.rb]: That's it :slightly_smiling_face:
